dev-db/percona-xtrabackup/percona-xtrabackup-2.4.5.ebuild

```diff
@@ -11,7 +11,7 @@ SRC_URI="https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-${PV}/s
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="amd64 x86"
```

Comment:
I am one of the maintainers https://github.com/gentoo/gentoo/blob/master/dev-db/percona-xtrabackup/metadata.xml#L9 

And we do have STABLEREQ bug: https://bugs.gentoo.org/show_bug.cgi?id=606110